chips/stm32h7/memory-large.toml

```diff
@@ -14,6 +14,14 @@ read = true
 write = true
 execute = false  # let's assume XN until proven otherwise
 
+# We use DTCM for a handoff region
```

Comment:
turbo nit: it might be nice if this comment explained a _bit_ more about what that meant, so that the reader knows where to go for more information?